spyderlib/widgets/comboboxes.py

```diff
@@ -195,7 +280,7 @@ class UrlComboBox(PathComboBox):
     def __init__(self, parent, adjust_to_contents=False):
         PathComboBox.__init__(self, parent, adjust_to_contents)
         self.editTextChanged.disconnect(self.validate)
-        

```

Comment:
Please remove blanks *only* if they're around the code you're touching. This is not a problem in this case because we almost never touch `comboboxes`, so it's a blocker for merging :-)

But I've had major headaches trying to merge things between 2.3 and master when that rule is not followed :-)

My idea is to (slowly) move to remove blanks and adhere to pep8, by following Raymond Hettinger advice: https://www.youtube.com/watch?v=wf-BqAjZb8M